### PR TITLE
contact fetch added, contact export added

### LINKED ISSUE #36: Update contact/export.sh
https://github.com/EasyGnuPG/pgpg/blob/d108baf68dc2eb6de3d360b4bb1e06f2017d0946/src/cmd/contact/export.sh#L25

create contact/export.py

### LINKED ISSUE #37: Update contact/fetch.sh
https://github.com/EasyGnuPG/pgpg/blob/d108baf68dc2eb6de3d360b4bb1e06f2017d0946/src/cmd/contact/fetch.sh#L33

https://github.com/EasyGnuPG/pgpg/blob/d108baf68dc2eb6de3d360b4bb1e06f2017d0946/src/cmd/contact/fetch.sh#L36

create contact/fetch.py

### PR DESCRIPTION
**Pull request by diveshuttam**
closes #37 closes #36

### CODE REVIEW

**Comment by dashohoxha** [comment]
Divesh, it is not a good practice to stack pull requests on top of each-other. Either continue pushing on the existing unmerged branch, or create a new branch from the base branch (in this case gnupg-2.2).

**Comment by diveshuttam** [comment]
Ok, I also generally avoid this but this feature depended on the contact export functionality so I had to checkout from that branch. And I created a different PR as this addressed a different issue.
But now I think I could have also referenced the issue in another commit within #76 .

**Comment by diveshuttam** [comment]
BTW thinking again. We can close #76 and just merge this as it addresses both the issues. 
I'll keep this in mind from next time.

File: src/cmd/contact/fetch.sh [hunk 1]
```diff
@@ -27,13 +27,20 @@ cmd_contact_fetch() {
     [[ -d "$homedir" ]] || fail "Cannot find gnupg directory: $homedir"
     echo "Importing contacts from: $homedir"
 
+    # save current GNUPGHOME and export new homedir
+    local GNUPGHOME_BAK=$GNUPGHOME
+    export GNUPGHOME=$homedir
```
Comment (dashohoxha) [inline src/cmd/contact/fetch.sh]: I would prefer to set the `$homedir` on `gpg.Context()`.

Comment (diveshuttam) [reply src/cmd/contact/fetch.sh]: I have currently done it for contact/export only as it wasn't needed anywhere for import.

**Review by dashohoxha — APPROVED** [review]
Seems OK.

**Pull request merged**